### PR TITLE
Feat/local dev setup

### PR DESCRIPTION
**Pull request by ciaranschutte**
**Description of changes**

Lots of small cleanup and a way to run locally with live reload.

- adds local development script
- updated readme
- consolidate `package.json` 
- keep `npmrc` from `platform`
- babel presets instead of importing `nextjs` just for its babel config
- some clean up of renaming and simplifying now that `uikit` is it it's own repo

**Type of Change**

- [ ] Bug
- [ ] Styling
- [ ] New Feature

**Checklist before requesting review:**

- Design (select one):
  - [ ] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [ ] No design provided, screenshot of changes included in PR
  - [ ] No changes to UI
- UI Kit (select one):
  - [ ] New Components with storybook stories created
  - [ ] Modified Existing components and updates stories
  - [ ] No new UIKit components or changes
- [ ] Feature is minimally responsive
- [ ] Manual testing of UI feature
- [ ] Add copyrights to new files
- [ ] Connected ticket to PR

### CODE REVIEW

**Comment by joneubank** [comment]
Pedantic maybe, but its been renamed to `readme.md` and should be standard uppercase `README.md` :)

**Comment by ciaranschutte** [comment]
> Pedantic maybe, but its been renamed to `readme.md` and should be standard uppercase `README.md` :)

pendantic is great to get this off on the correct footing.
thinking I may even change the repo name to @icgc-argo/argo-uikit but the redundancy hurts me a little

**Comment by joneubank** [comment]
> thinking I may even change the repo name to @icgc-argo/argo-uikit but the redundancy hurts me a little
Please no... the useless repitition in all the other repos hurts me. daily.

**Comment by ciaranschutte** [comment]
build issues fixed in https://github.com/icgc-argo/uikit/pull/9

File: README.md [hunk 1]
```diff
@@ -45,31 +50,15 @@ To keep commit messages consistent, we use [gitmoji-cli](https://www.npmjs.com/p
...
+  2. Merge that change to `develop`, and let the Jenkins pipeline handle it
 
-    2. Run `npm run build-uikit` which performs the following:
-    3. Run `npm run publish-uikit` to publish latest version to npm
+- Manual process (discouraged, but available)
 
-- ### Restriction:
-  - `uikit` components should not reference (`import`) anything outside of the `uikit` directory
+  2. Run `npm run build-uikit` which performs the following:
+  3. Run `npm run publish-uikit` to publish latest version to npm
```
Comment (joneubank) [inline README.md:64]: Thanks for details!

**Review by joneubank — APPROVED** [review]
Big thanks for following through with this, and extra credit for the README develop notes for both UIKit and Integration with other projects.
🎉 🎉 🎉 🎉 🎉

**Pull request merged**